docs/configs-npb-tests/run_npb.py

```diff
@@ -52,52 +47,33 @@
 
 from system import MySystem
 
-SimpleOpts.add_option("--benchmark", default='',
-                      help="Script to execute in the simulated system")
-SimpleOpts.add_option("--benchmark_inputs", default='',
-                      help="Script to execute in the simulated system")
+def writeBenchScript(dir, bench):
+    """
+    This method creates a script in dir which will be eventually
+    passed to the simulated system (to run a specific benchmark
+    at bootup).
+    """
+    bench_file = open('{}/run_{}'.format(dir, bench),"w+")
+    bench_file.write('/home/gem5/NPB3.3-OMP/bin/{} \n'.format(bench))
+    bench_file.write('m5 exit \n')
+    bench_file.close()
+    return bench_file.name
```

Comment:
I'm surprised this works. It looks like only a "raw file" has a name member: https://docs.python.org/3/library/io.html#raw-file-i-o. Though, I admit I can't quite follow the inheritance tree in that documentation.

I suggest making a new local variable (e.g., `file_name`) and using that on line 56 and returning that.

Reply:
updated